<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerPage</class>
 <widget class="QWidget" name="ServerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>594</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Identity:</string>
       </property>
       <property name="buddy">
        <cstring>identitySelector</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="identitySelector"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Networks</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="KListWidget" name="networkList">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="currentRow">
               <number>-1</number>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="KPushButton" name="addNetworkButton">
                <property name="text">
                 <string>Add Network...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KPushButton" name="editNetworkButton">
                <property name="text">
                 <string>Edit Network...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KPushButton" name="deleteNetworkButton">
                <property name="text">
                 <string>Delete Network</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KPushButton" name="importNetworks">
                <property name="text">
                 <string>Import Networks...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KPushButton" name="exportNetworks">
                <property name="text">
                 <string>Export Networks...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="KTabWidget" name="serverTabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Connection</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0" colspan="2">
                 <widget class="QGroupBox" name="useReconnection">
                  <property name="title">
                   <string>Reconnection</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="0" column="0">
                    <layout class="QFormLayout" name="formLayout_3">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_2">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string>Retry interval (seconds):</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="buddy">
                        <cstring>retryInterval</cstring>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="KIntSpinBox" name="retryInterval">
                       <property name="suffix">
                        <string> s</string>
                       </property>
                       <property name="minimum">
                        <number>-1</number>
                       </property>
                       <property name="maximum">
                        <number>30</number>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Retry attempts:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="buddy">
                        <cstring>retryAttempts</cstring>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="KIntSpinBox" name="retryAttempts"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>Connection Options</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_6">
                   <item row="0" column="0">
                    <layout class="QFormLayout" name="formLayout_4">
                     <item row="0" column="0" colspan="2">
                      <widget class="QCheckBox" name="connectOnStartup">
                       <property name="text">
                        <string>Connect on startup</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0" colspan="2">
                      <widget class="QCheckBox" name="useSSL">
                       <property name="text">
                        <string>Use SSL</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Servers</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_5">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="1" column="0" colspan="2">
                 <widget class="QCheckBox" name="connectToRandomServer">
                  <property name="text">
                   <string>Connect to random server</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="KEditListBox" name="serverList">
                  <property name="title">
                   <string>Servers</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Channels</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_6">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0" colspan="2">
                 <widget class="KEditListBox" name="channelList">
                  <property name="title">
                   <string>Channels</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QCheckBox" name="autoJoinChannels">
                  <property name="text">
                   <string>Automatically join channels</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Encoding</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_7">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0" colspan="2">
                 <widget class="QGroupBox" name="useDefaultEncoding">
                  <property name="title">
                   <string>Default Encoding</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_10">
                   <item row="0" column="0">
                    <layout class="QFormLayout" name="formLayout_8">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string>Default encoding:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="KComboBox" name="defaultEncoding"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Identify</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_9">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0" colspan="2">
                 <widget class="QGroupBox" name="useAutoIdentify">
                  <property name="title">
                   <string>Auto Identify</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_12">
                   <item row="0" column="0">
                    <layout class="QFormLayout" name="formLayout_10">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Service name:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="buddy">
                        <cstring>serviceName</cstring>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="KLineEdit" name="serviceName">
                       <property name="showClearButton" stdset="0">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Service password:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="buddy">
                        <cstring>servicePassword</cstring>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="KLineEdit" name="servicePassword">
                       <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                       </property>
                       <property name="urlDropsEnabled">
                        <bool>false</bool>
                       </property>
                       <property name="showClearButton" stdset="0">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
